cmd/brew-bootstrap-rbenv-ruby

```diff
@@ -11,6 +11,35 @@ fi
 warn() { echo "$@" >&2; }
 abort() { EXPECTED_EXIT="1"; warn "$@"; exit 1; }
 
+warn_sh() {
+  abort 'Error: add `eval "$(rbenv init -)"` to the end of your .bash_profile!'
+}
+
+warn_zsh() {
+  abort 'Error: add `eval "$(rbenv init -)"` to the end of your .zshrc!'
+}
+
+warn_fish() {
```

Comment:
If these functions abort, should they be called `warn_`? Seems like `abort_for_$shell`'d be a clearer name.